src/coprocessor/codec/error.rs

```diff
@@ -148,4 +152,11 @@ impl From<io::Error> for Error {
     }
 }
 
+impl From<RegexpError> for Error {
+    fn from(err: RegexpError) -> Error {
+        let msg = format!("Got error '{:.64}' from regexp", err.description());
```

Comment:
@AndreMouche Shall we unify the error message with MySQL?

Reply:
Just try to unify the error message with TIDB, not must.